docker/docker-entrypoint.sh

```diff
@@ -6,11 +6,8 @@ psql -h db -U postgres -c "CREATE ROLE onadata WITH SUPERUSER LOGIN PASSWORD 'on
 psql -h db -U postgres -c "CREATE DATABASE onadata OWNER onadata;"
 psql -h db -U postgres onadata -c "CREATE EXTENSION postgis; CREATE EXTENSION postgis_topology;"
 
-virtualenv /srv/onadata/.virtualenv
-. /srv/onadata/.virtualenv/bin/activate
```

Comment:
I would maintain the virtualenv within the source code directory, I wanted to share the same virtual environment that I could access when not using the docker container and that meant persisting it outside of the container. The alternative I see would be providing defining it as a volume, in either case, this may need to be initialized during startup of the container, hence it's place within the `docker-entrypoint.sh`.